assets/src/design-system/components/menu/list/components.js

```diff
@@ -43,36 +43,19 @@ export const ListItemLabel = styled.li`
 `;
 
 export const ListItem = styled.li(
-  ({ disabled, isSelected, theme }) => css`
+  ({ disabled, theme }) => css`
     display: grid;
     grid-template-columns: 16px 1fr;
     padding: 6px 2px 6px 8px;
     margin: 4px 8px;
     border-radius: ${theme.borders.radius.small};
     align-items: center;
-    background-color: ${isSelected
```

Comment:
These colors were adjusted in figma this week to make the new designs less purple 🟣 💜 🟪